### DISCUSSION TITLE
⭐️ Pake 共建计划

### DISCUSSION THREAD

**Discussion by tw93** [tw93/Pake#76] [Announcements]
## 缘起
> 业余时间在开源妙言的主线上，起了两个支线任务，一个是 Pake，一个是潮流周刊，前者增长速度突破我的想象，好几次在 Github Rust 日榜首，5天 Star 就到了1.5K，比妙言猛了不少，但难度不到妙言的 1/10，适合开源新手，有不少可以做的事情，由于后面业余时间有限，重点还是弄妙言，期待可以每月一个新版本，Pake 也很有搞头，欢迎大家参与，参与有价值 PR 超过5次，可以加你成为开发者。

## Todolist
- [x] 1、代码架构优化，可以将注入的 css、注入的 js 通过文件方式分开，同时所有配置可以放到 tauri.config.json 里面
- [x] 2、使用 Github Actions 支持 Pake 自动打常用包提交到 Release
- [x] 3、支持 Windows 的打包，可以和其他项结合起来 
- [x] 4、通过命令行 Pake 可以直接生成指定包，无需 fork 仓库的那种，类似 https://github.com/nativefier/nativefier 
- [ ] 5、搭建简单官网，支持常用应用的分系统下载和预览
- [x] 6、回复issue，解决用户疑问

## 领取
- 对于想支持的同学，可以直接领取一个 Todo，你作为负责人，我这边来一个 Pake 开发群到时候讨论即可。
- 可以直接回复 Issue 进行报名
- 开源过程是一件很有趣的事情，我 10 年前大学刚开始学习技术的时候，超级感谢 github 的帮助，请自由地享受和参与开源。

## 最近更多在做的
- https://github.com/tw93/Pake/issues/112
- https://github.com/tw93/Pake/issues/110
- https://github.com/tw93/Pake/issues/98

## 当前所有的PR
- https://github.com/tw93/Pake/pulls?q=

**Comment by xiamu14** [comment]
我可以领取 5 搭建官网任务。我作为前端开发已经工作 5年了，因为目前刚开始学习 rust ，但可以从搭建官网开始参与项目共建，之后也许可以参与开发更核心的功能。

**Comment by wanghanzhen** [comment]
认领4

**Comment by jokereven** [comment]
认领6

**Comment by piaoyidage** [comment]
认领2

**Comment by houseme** [comment]
小伙伴们杠杠的

**Comment by tw93** [comment]
@houseme 你要不要试试 3

**Comment by houseme** [comment]
> @houseme 你要不要试试 3

给其他小伙伴吧

**Comment by charlzyx** [comment]
我试试3

**Reply by yeyouchuan** [reply]
求一个 readwise reader 的 windows 打包，我可以提供内测邀请！

**Comment by timeTravelCYN** [comment]
已经被领完了～还有 todo 吗

**Comment by tw93** [comment]
@timeTravelCYN 搭建官网以及回答issue这一块，其实可以多人的

**Comment by timeTravelCYN** [comment]
> @timeTravelCYN 搭建官网以及回答issue这一块，其实可以多人的

举手官网

**Comment by tw93** [comment]
@charlzyx @timeTravelCYN 辛苦加一下我的tg号，我拉你两进群

**Comment by charlzyx** [comment]
> @charlzyx @timeTravelCYN 辛苦加一下我的tg号，我拉你两进群

尴尬， 还没看到tg 回复，现学rust 几天瞎比划了一下 windows 的， 有冲突， 暂时不合并， 看下 diff 吧

https://github.com/tw93/Pake/compare/master...charlzyx:Pake:master

![image](https://user-images.githubusercontent.com/18055018/197823735-ae988c65-465f-4735-a0d4-3d58dcf55016.png)

https://qwerty.kaiyi.cool/gallery

**Comment by yuxiaoy1** [comment]
> @timeTravelCYN 搭建官网以及回答issue这一块，其实可以多人的

能否拉我下，想参与搭建官网

**Comment by tw93** [comment]
@yuxiaoy1 很欢迎哈，你把你的tg号给我一下，我加你

**Comment by yuxiaoy1** [comment]
yuxiaoy

**Comment by daodaolee** [comment]
一个前端开发者，推上看到pake，目前vue多一点，rust处于刚了解阶段，期待共建😉，我的社交信息都在这里@daodaolee

**Reply by tw93** [reply]
赞赞赞，可以先去看看pake的整体，看看有没有想优化的，欢迎直接来弄

**Reply by daodaolee** [reply]
好哒！

**Comment by anguschiu1** [comment]
很少見 open source project 剛開始的 docs 就那麼好，tree structure, usage, contribution guideline 一應俱全⋯⋯厲害！

**Comment by elvinzou** [comment]
Pake 的小红书登录后很容易违规，小红书会冻结账号，不知道是不是只有我这样

**Comment by Samuelonly** [comment]
<img width="1200" height="800" alt="Think fast，" src="https://github.com/user-attachments/assets/cde4709c-e281-4132-937c-a9e83058b264" />

<img width="1200" height="800" alt="xw_20260312152015" src="https://github.com/user-attachments/assets/6e55c31f-3c8b-4cc2-b496-63c50d1d10f8" />

<img width="1200" height="800" alt="xw_20260312162513" src="https://github.com/user-attachments/assets/44a177c2-6d2e-4b65-9721-a8c76a067bc3" />


<img width="1554" height="712" alt="xw_20260312165003" src="https://github.com/user-attachments/assets/6caa1c2a-c1bf-488e-b62e-df189107a6aa" />
